### PR TITLE
Adding class type into the PR and F1 scores 

### PR DESCRIPTION
**Pull request by axiomcura**
This PR adds the injury type levels to the f1 score and PR scores

### CODE REVIEW

File: notebooks/2.modeling/nbconverted/2.modeling.py [hunk 1]
```diff
@@ -182,14 +212,15 @@
 
 # ### Evaluating Multi-class model trained with original split with holdout data
 
-# In[ ]:
+# In[12]:
 
 
 # evaluating with plate holdout
 plate_ho_precision_recall_df, plate_ho_f1_score_df = evaluate(
     model=best_model,
```
Comment (MikeLippincott) [inline notebooks/2.modeling/nbconverted/2.modeling.py:220]: I noticed these X data sets are not shuffled?

File: notebooks/2.modeling/nbconverted/2.modeling.py [hunk 2]
```diff
@@ -208,6 +240,7 @@
...
```
Comment (MikeLippincott) [inline notebooks/2.modeling/nbconverted/2.modeling.py:240]: same here.

**Review by MikeLippincott — APPROVED** [review]
I am not sure if there needs to be a shuffled dataset also for the holdout models. Other than that LGTM

File: notebooks/2.modeling/nbconverted/2.modeling.py (hunk 1 above)
Comment (axiomcura) [reply notebooks/2.modeling/nbconverted/2.modeling.py:220]: Hey @MikeLippincott! the shuffled data is shown [here](https://github.com/WayScience/Cytotoxic-Nuisance-Metadata-Analysis/blob/d3984e413cb9127db95c8a69de8ee876bc87ed33/notebooks/2.modeling/nbconverted/2.modeling.py#L143-L153)

File: notebooks/2.modeling/nbconverted/2.modeling.py (hunk 2 above)
Comment (axiomcura) [reply notebooks/2.modeling/nbconverted/2.modeling.py:240]: same response above

**Comment by axiomcura** [comment]
Thank you for the review @MikeLippincott, merging now!

**Pull request merged**